services/datamanager/builtin/sync_test.go

```diff
@@ -320,13 +330,14 @@ func TestDataCaptureUpload(t *testing.T) {
 // Cases: manual sync, successful, unsuccessful
 // To validate: uploaded all data once, deleted file if no error, builds additional sync paths if none exist
 func TestArbitraryFileUpload(t *testing.T) {
+	t.Skip()
 	// Set exponential factor to 1 and retry wait time to 20ms so retries happen very quickly.
 	datasync.RetryExponentialFactor.Store(int32(1))
 	datasync.InitialWaitTimeMillis.Store(int32(20))
 	syncTime := time.Millisecond * 100
 	datasync.UploadChunkSize = 1024
 	fileName := "some_file_name.txt"
-	fileExt := ".txt"
+	// fileExt := ".txt"
```

Comment:
I know we're skipping this for now, but there are quite a few LOC commented out in this test function. Did you mean to do that? It looks like its because we're using the mock grpc service, which we got rid of. Maybe we can at least add a comment explaining why this is commented out for now and a link to a ticket that will resolve this?

Reply:
Added a comment with a TODO linking [this ticket](https://viam.atlassian.net/browse/DATA-1268) that I'll pick up right after this

Reply:
Thank you!